cmd/ethatomicswap/main.go

```diff
@@ -0,0 +1,1364 @@
+// Copyright (c) 2018 The Decred developers and Contributors
```

Comment:
Remove "and Contributors" and add additional copyright below for yourself

Reply:
You do want the decred developers here? All good for me though, just want to be clear where I do need to add decred copyright, if in any `ethatomicswap` related file at all.

Reply:
For this file, yes.  It includes code copied from the other tools and must retain the Decred developers copyright.